index.d.ts

```diff
@@ -6,6 +6,13 @@ declare namespace cpFile {
 		@default true
 		*/
 		readonly overwrite?: boolean;
+
+		/**
+		Perform a fast copy-on-write clone (reflink) if possible.
+
```

Comment:
Not clear from the docs here what `force` does.

Maybe also mention some use-cases for needing to set it to `false` or `'force'` as I cannot think of anything reason to not have this be just `true` all the time.

Reply:
This is analogous to gnu cp's `--reflink=auto` vs `--reflink`. From a usability standpoint, it is usually preferable to have a fallback (hence `true`), but when someone comes onto a new platform it's nice to have a test for whether reflink is supported (hence `"force"`)